README.md

```diff
@@ -142,34 +142,127 @@ Parse Dashboard is continuously tested with the most recent releases of Node.js
+| `secondaryBackgroundColor` | String              | yes      | -         | -                    | -                                    | `"#FF4500"`                       | Secondary background color (CSS value).                                                                      |
+| `supportedPushLocales`     | Array&lt;String&gt; | yes      | -         | -                    | -                                    | `["en","fr"]`                     | Supported locales for push notifications.                                                                    |
+| `preventSchemaEdits`       | Boolean             | yes      | `false`   | -                    | -                                    | `true`                            | Prevent schema modifications through the dashboard.                                                          |
+| `columnPreference`         | Object              | yes      | -         | -                    | -                                    | `{"_User":[...]}`                 | Column visibility/sorting/filtering preferences. See [column preferences details](#prevent-columns-sorting). |
+| `classPreference`          | Object              | yes      | -         | -                    | -                                    | `{"_Role":{...}}`                 | Persistent filters for all users. See [persistent filters details](#persistent-filters).                     |
+| `enableSecurityChecks`     | Boolean             | yes      | `false`   | -                    | -                                    | `true`                            | Enable security checks under App Settings > Security.                                                        |
+| `cloudConfigHistoryLimit`  | Integer             | yes      | `100`     | -                    | -                                    | `200`                             | Number of historic Cloud Config values (0 to Number.MAX_SAFE_INTEGER).                                       |
+| `config`                   | Object              | yes      | -         | -                    | -                                    | `{...}`                           | Settings for storing dashboard config on server.                                                             |
+| `config.className`         | String              | yes      | -         | -                    | -                                    | `"DashboardConfig"`               | Table name for dashboard configuration.                                                                      |
+| `scripts`                  | Array&lt;Object&gt; | yes      | `[]`      | -                    | -                                    | `[{...}]`                         | Scripts for this app. See [scripts table below](#scripts).                                                   |
```

Comment:
Could you add a root table at the top that describes the root keys with links like here?